CHANGELOG.md

```diff
@@ -6,38 +6,40 @@ _For older release notes, see_ [PWA Studio releases][].
 
 ## New Features
 
--  **Added PWA Studio metapackages** — In this release, we introduce our PWA Studio metapackages: One for [Magento Open Source](https://github.com/magento-commerce/magento2-pwa) and one for [Adobe Commerce](https://github.com/magento-commerce/magento2-pwa-commerce). These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We added GraphQL endpoints and fields to the metapackages to fix some `Cart`-related bugs. GitHub PRs: [4][], [1][].
+-  **Added PWA Studio metapackages** — In this release, we introduce our PWA Studio metapackages: One for [Magento Open Source](https://github.com/magento/magento2-pwa) and one for [Adobe Commerce](https://github.com/magento/magento2-pwa-commerce). These metapackages provide a new way to add any additional Open Source or Commerce features needed by your PWA modules. The PWA Studio team will also use these metapackages to add new features to the Open Source and Commerce code bases as needed. In fact, we did that in this release. We added a GraphQL mutation and several fields to the metapackages to fix some `Cart`-related bugs. GitHub PRs: [4][], [1][].
```

Comment:
Sorry about the churn. Change the last sentence to say something like

We extended the GraphQL schema to include a new mutation and to provide more details when a cart item error occurs.

Currently, you don't provide any details about https://github.com/magento-untitled-geese/magento2-pwa/pull/2/files, so being vague at the top lets you dive into detail below without repeating yourself.